app/src/main/java/org/simple/clinic/selectstate/SelectStateUiRenderer.kt

```diff
@@ -10,6 +10,11 @@ class SelectStateUiRenderer(
 ) : ViewRenderer<SelectStateModel> {
 
   override fun render(model: SelectStateModel) {
+    if (model.isFetching) {
+      ui.showProgress()
+      return
```

Comment:
I prefer to use `if-else` blocks instead of early returns. Seems more readable to me. What do you think?

Reply:
Yeah, we can do that. Personally I prefer this, but `if-else` also makes sense